crates/types/src/config/common.rs

```diff
@@ -93,7 +92,7 @@ pub struct CommonOptions {
     /// Can be configured using the [`humantime`](https://docs.rs/humantime/latest/humantime/fn.parse_duration.html) format.
     #[serde_as(as = "serde_with::DisplayFromStr")]
     #[cfg_attr(feature = "schemars", schemars(with = "String"))]
-    pub shutdown_timeout: Duration,
+    pub shutdown_timeout: humantime::Duration,
```

Comment:
Note: Probably this breaks config backward compatibility.  Perhaps split this into its own PR and validate it.

Reply:
It is still `humantime::Duration` because I only removed the prefix which was explicitly imported. This is actually confusing. Let me change it back to keep the `humantime` module.

Reply:
Yes, you are right. I will revert this change. Not sure what I thought about it (mostly making it consistent).

Reply:
Good catch!

Reply:
Actually, I think it was before also `humantime::Duration`, I just wanted to make it explicit that this is not `std::time::Duration`.